recipes/_configure_server.rb

```diff
@@ -133,33 +133,49 @@
 end
 
 # License Configuration
+node.run_state['cerner_splunk'] ||= {}
+
 license_uri =
   case node['splunk']['node_type']
-  when :forwarder, :license_server
+  when :license_server
     'self'
   when :cluster_master, :cluster_slave, :server, :search_head, :shc_search_head, :shc_captain, :shc_deployer
     if node['splunk']['free_license']
       'self'
     else
       CernerSplunk.my_cluster_data(node)['license_uri'] || 'self'
     end
+  when :forwarder
+    if node.run_state['cerner_splunk']['heavy_forwarder']
+      CernerSplunk.my_cluster_data(node)['license_uri'] || 'self'
```

Comment:
For now, I'm went with the boolean attribute to decide if we want to set to the license_uri or to _self_. 53686564d45fdf5133ee5bd8d780a694d63101d1